apps/web/src/components/Composer/LinkPreviews.tsx

```diff
@@ -8,7 +8,15 @@ import { usePublicationAttachmentStore } from 'src/store/non-persisted/publicati
 import { usePublicationAttributesStore } from 'src/store/non-persisted/publication/usePublicationAttributesStore';
 import { usePublicationStore } from 'src/store/non-persisted/publication/usePublicationStore';
 
-const LinkPreviews: FC = () => {
+interface LinkPreviewProps {
+  openActionEmbed: boolean;
+  openActionEmbedLoading: boolean;
```

Comment:
This Open action seems more specific to nft open action and not general OA

So instead os just naming openAction we can be specific to nftOpenAction,

I'm just worried about using link previews here, instead we can use `OpenActionOnBody` component https://github.com/heyxyz/hey/blob/main/apps/web/src/components/Publication/PublicationBody.tsx#L118 which we use it for swap RN